src/Microsoft.DotNet.Tools.Compiler.Native/IntermediateCompilation/Linux/LinuxCppCompileStep.cs

```diff
@@ -87,10 +87,11 @@ private void InitializeArgs(NativeCompileSettings config)
             }
 
             // AppDep Libs
+            var baseAppDeplibPath = Path.Combine(config.AppDepSDKPath, "CPPSdk/ubuntu.14.04/x64");
```

Comment:
That way we would end up with x86/x64/arm/arm64 x Ubuntu/OSX/Windows number of folders. I guess this is better and the AppDep SDK is somewhat a temporary story -- the layout here is more C++ specific than they are nuget specific.